sentry2csv/sentry2csv.py

```diff
@@ -12,6 +12,8 @@
 import aiohttp
 import pkg_resources  # part of setuptools
 
+from .constant import SENTRY_HOST
```

Comment:
I get where you're going with this, but, for now, could we just inline the constant? i.e., `SENTRY_HOST = "sentry.io"`